packages/block-library/src/quote/style.scss

```diff
@@ -18,4 +19,8 @@
 			text-align: right;
 		}
 	}
+
+	&.has-background {
+		padding: $block-bg-padding--v $block-bg-padding--h;
```

Comment:
We need to decide what padding value to use by default for quotes with background colors. We have several options here:

 - Match the paragraph/group and all similar blocks (like currently done in the PR)
 - Only add vertical padding because the quote comes with a horizontal padding by default
 - No padding at all.

Reply:
I'd suggest doing nothing. People can add padding if needed

Reply:
I'm AFK this week so can't take a close look at this PR just yet, but +1 for doing nothing and adding in the padding block support to handle it. I think from memory, with most of the other blocks where we've added padding support, we've had to maintain the CSS rule for backwards compatibility (e.g. behaviour from before adding the padding support) rather than as a desired bit of CSS to ship to the front end.

Reply:
Removed the padding.